project-koku/koku
# updating sources api error responses

The Sources API proxy endpoints were not responding proper json error objects and was making the integration with the UI unnecessarily complicated.

FYI: @rvsia

**Testing**
1.  Add a bucket to a non-existent source.  Verify error response is expected. {"errors": [{...}]}
2.  Add AWS source (source_id 1) and re-add bucket to verify the response is json
3. Add a subscription-id to a non-existent Azure source
4.  Add a subscription_id with malformed data

**Test Results**
[sources_api_error_response_ut.txt](https://github.com/project-koku/koku/files/3833130/sources_api_error_response_ut.txt)
